### PR TITLE
walletrpc: expose dry run support for ImportAccount

### PR DESCRIPTION
**Pull request by wpaulino**
This allows users to confirm whether their account is being imported with the correct parameters, and additionally returns the first 5 external and internal addresses for said account to confirm whether the address derivation used is correct.

As part of updating the btcwallet dependency, a series of changes were required to our protobuf generations due to a protobuf version bump. These changes were cherry picked from commit https://github.com/lightningnetwork/lnd/pull/4998/commits/e5f249ad51bf7dcec1861d837367fcdd82574f31 in https://github.com/lightningnetwork/lnd/pull/4998.

Depends on https://github.com/btcsuite/btcwallet/pull/746.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -249,8 +249,8 @@ jobs:
       fail-fast: false
       matrix:
         pinned_dep:
-          - google.golang.org/grpc v1.24.0
-          - github.com/golang/protobuf v1.3.2
+          - google.golang.org/grpc v1.29.1
```
Comment (Roasbeef) [inline .github/workflows/main.yml:252]: Can be removed now as this has landed in master.

File: lnwallet/btcwallet/btcwallet.go [hunk 2]
```diff
@@ -459,12 +459,39 @@ func (b *BtcWallet) ListAccounts(name string,
...
-	_, err := b.wallet.ImportAccount(
-		name, accountPubKey, masterKeyFingerprint, addrType,
+	if !dryRun {
+		accountProps, err := b.wallet.ImportAccount(
+			name, accountPubKey, masterKeyFingerprint, addrType,
+		)
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		return accountProps, nil, nil, err
```
Comment (Roasbeef) [inline lnwallet/btcwallet/btcwallet.go:473]: Last return value can just be `nil`.

File: lnwallet/btcwallet/btcwallet.go [hunk 3]
```diff
@@ -459,12 +459,39 @@ func (b *BtcWallet) ListAccounts(name string,
...
+	}
+
+	// Generate 5 addresses by default for the dry run.
+	accountProps, extAddrs, intAddrs, err := b.wallet.ImportAccountDryRun(
+		name, accountPubKey, masterKeyFingerprint, addrType, 5,
```
Comment (Roasbeef) [inline lnwallet/btcwallet/btcwallet.go:478]: Should move the `5` to a constant somewhere, likely "good enough" for now as a base value.

**Review by Roasbeef — COMMENTED** [review]
Solid use-ability improvements! Just a few style nits on my end

**Review by Roasbeef — APPROVED** [review]
LGTM 🧮

**Review by guggero — APPROVED** [review]
Very useful feature! tACK, LGTM :100:

**Pull request merged**